src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
```

Comment:
Hi @DamnedScholar,

Sorry for keeping this in the *limbo*, do you want me to accept this and make the necessary changes for this to work as we discussed?

By all means I want to take your work, in fact, I want to thank you for picking this up and allowing us to reach a nice workflow.